x-pack/plugins/ingest_pipelines/public/application/components/pipeline_editor/__jest__/processors/bytes.test.tsx

```diff
@@ -85,7 +79,47 @@ describe('Processor: Bytes', () => {
     const processors = getProcessorValue(onUpdate, BYTES_TYPE);
     expect(processors[0].bytes).toEqual({
       field: 'field_1',
-      ...defaultBytesParameters,
+    });
+  });
+
+  test('saves with common fields set', async () => {
+    const {
+      actions: { addProcessor, saveNewProcessor, addProcessorType },
+      form,
+      find,
+    } = testBed;
+
+    // This test ensures that the common fields that are used across all processors
+    // works and removes the need for those fields to be in every processors' test.
+
+    // Open flyout to add new processor
+    addProcessor();
+    // Add type (the other fields are not visible until a type is selected)
+    await addProcessorType(BYTES_TYPE);
+    // Add "field" value (required)
+    form.setInputValue('fieldNameField.input', 'field_1');
+
+    form.toggleEuiSwitch('ignoreFailureSwitch.input');
+
+    form.setInputValue('targetField.input', 'target_field');
```

Comment:
`targetField` is not common for all processor types. I would remove this line based on the test description.

Reply:
@alisonelizabeth Thanks for clarifying that. I used our folder structure as a little bit of a guide and noticed that the tag field was in the common field types so I included that. Removing. 
<img width="353" alt="Screen Shot 2021-05-06 at 12 23 31 PM" src="https://user-images.githubusercontent.com/1432078/117333232-b74b6180-ae66-11eb-87d7-bf1bff51a605.png">

Reply:
👍 It's applicable to some processors, but not all 😄. That's probably why it's in the `common_fields` directory. I think the only fields that are common for *all* are the fields found in the component called `common_processors.tsx` - `if`, `ignore_failure` and `condition`.